product_docs/docs/migration_toolkit/55/07_invoking_mtk/mtk_command_options_in_file/executing_migration_with_txt.mdx

```diff
@@ -0,0 +1,87 @@
+---
+title: "Executing a migration with a command options file"
+navTitle: "Executing a migration with the options file"
+deepToC: true
+---
+
+After you have created the options file, execute the migration command by referencing the options file:
+
+```shell
+./runMTK.sh -optionsFile <example.options>
+```
+
+!!!note
+   Provide the full path for `<example.options>` if you are running the MTK command from a different folder than where you have stored the file.
+!!!
+
+You can specify all options in the file, or use a mix of command line and options file to specify the migration parameters.
+
+Here are some examples for specifying all options in the file:
+
+## Example 1
+
```

Comment:
I would label the example. 
For example, 

> Migrating a specific schema specifying all options via the command line

Reply:
Also, it would be good to have a short statement indicating that when migrating only one or a subset of schemas, the schema name(s) must be listed at the end of the command. Migration Toolkit does not currently support specifying a list of schemas via an option file.  Perhaps this should be pointed out in one of the other pages, but for now, it would be acceptable to make that clear here.